src/components/EventCard/EventCard.tsx

```diff
@@ -77,6 +81,12 @@ const EventCard = ({
                 imageAltTag={`image for ${maker}`}
               />
             )}
+            <div className="event-card__capacity">
+              <p className="event-card__capacity-text">
+                There are {capacityMax - capacityCurrent} spaces left out of{" "}
+                {capacityMax}
+              </p>
+            </div>
           </div>
```

Comment:
this should be a conditional render for example. If the event is after todays date then it should say something like the attendance of this event is X or you can have how many spaces are available.

Reply:
Done!